docs/user-guide/connectors/custom-connectors.rst

```diff
@@ -31,6 +31,17 @@ To send a message, you would run a command like:
 
 where ``myio`` is the name of your component.
 
+You can also send extra information from your front-end to your
+action server, with it you can use, e.g. name, locations, or tokens
```

Comment:
@erohmensing Actually to the input channel and from there it goes to the action server? Can we somewhat clarify that part?

Reply:
Haha yes this was just a rough idea of what it might sound like 😃How about _something like_ "If you need to use extra information from your front end in your custom actions, you can add this information in the `metadata` dict of your usermessage. This information will accompany the user message through the rasa server into the action server when applicable, where you can use it via the <method to get last message's metadata method here> method. This information will not directly affect the NLU classifcation or action prediction.